index.html

```diff
@@ -2,9 +2,135 @@
+    <title>Lesson 3 homework</title>
 </head>
 <body>
+<img src="https://www.history.com/.image/ar_1:1%2Cc_fill%2Ccs_srgb%2Cfl_progressive%2Cq_auto:good%2Cw_1200/MTk1NzY0MDg3NTEzMTYzNDEy/gettyimages-142198198.jpg"
+     width="50%" alt="Eiffel Towe">
+<ol>
+    <li>Знакомство с Web</li>
+    <ul>
+        <li>Краткая история Web и что такое интернет</li>
+        <li>Как устроен браузер и как он получает страницы по сети</li>
+        <li>Что такое Git?</li>
+        <li>Создание первого репозитория</li>
+        <li>О файле .gitignore</li>
+    </ul>
 
+    <li>HTML-верстка- первые шаги</li>
+    <ul>
+        <li>Настройка рабочей среды</li>
+        <li>Краткая история HTML</li>
+        <li>Какие бывают теги и что такое атрибуты</li>
+        <li>Теги для работы с текстом</li>
+        <li>Создание собственной страницы</li>
+    </ul>
+
+    <li>Погружение в HTML-разметку</li>
+    <ul>
+        <li>Мета-теги</li>
+        <li>Мнемоники</li>
+        <li>Тег для работы с изображениями</li>
+        <li>Теги для работы со списками</li>
+        <li>Теги для работы с таблицами</li>
+        <li>Теги для работы с формами</li>
+        <li>Div-блоки</li>
+    </ul>
+</ol>
+<table border="1" width="100%">
+    <tr>
+        <th>Tag</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>table</td>
+        <td>Defines a table</td>
+    </tr>
+    <tr>
+        <td>tr</td>
+        <td>Defines a row in table</td>
+    </tr>
+    <tr>
+        <td>th</td>
+        <td rowspan="2">Defines a "header cell"/"cell" in a table</td>
+    </tr>
+    <tr>
+        <td>td</td>
+    </tr>
+    <tr>
+        <td colspan="2">Rarely used</td>
+    </tr>
+    <tr>
+
+        <td>caption</td>
+        <td>Defines a table caption</td>
+    </tr>
+    <tr>
+
+        <td>thead</td>
+        <td>Groups the header content in a table</td>
+    </tr>
+    <tr>
+        <td>tbody</td>
+        <td>Groups the body content in a table</td>
+    </tr>
+    <tr>
+        <td>tfoot</td>
+        <td>Groups the footer content in a table</td>
+    </tr>
+</table>
+
+<p>
+<form action="/" method="GET">
+    <fieldset>
+        <legend>My first form</legend>
+        <label for>Name</label><br>
+        <label for="name"></label><input id="name" type="text" name="name" placeholder="Name" required><br>
+        <label for>Email</label><br>
+        <label for="email"></label><input id="email" type="email" name="email" required><br>
+        <label for>Password</label><br>
+        <label for="Password"></label><input id="Password" type="text" name="password" required><br>
+        <label for="level">Your programming level</label><br>
+        <input id="level" type="range" name="level" min="0" max="10">
+
+        <p>
+            <label>What does HTML stand for?</label>
+        </p>
+        <label id="html-1">Hyperlinks and Text Markup Language</label><br>
+        <input id="html" type="checkbox" value="1">
```

Comment:
There can be only one tag on a page with a unique id , but your id is repeated, this is a bit wrong) if you want to make many +- identical ids, make html1, html2, etc.